funnel/models/space.py

```diff
@@ -109,24 +120,26 @@ def proposals_all(self):
             return self.proposals
 
     @property
-    def proposals_by_status(self):
-        from .proposal import Proposal, PROPOSALSTATUS
+    def proposals_by_state(self):
+        from .proposal import Proposal
```

Comment:
Is it perhaps to better to move this method to the `Proposal` model so you don't have to import inside the function? Especially since it's proposals that are being queried for anyway.

Reply:
There are 3 methods - `proposals_all`, `proposals_by_state` and `proposals_by_confirmation`. I can move all 3 to the Proposal model if needed. At first I thought it makes sense to have them here as proposals have backreferences to the proposal spaces, but we don't seem to be using backreferences for these queries. in this case we can move them to Proposal model. But should I do that in this PR or create a new PR for that? Because it's going to be another level of refactoring unrelated to this PR.